README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Contributing
 
-### Local dev
+### Local dev (Experimental)
```

Comment:
Note: I marked as experimental as I cannot get to work properly. Josh and I spoke about this and we'd have to make so changes to the `server` file for ports perhaps and were concerned that testing in dev may now be the same experience as packaging up and running. But, TBD.

Reply:
Fine for now. But I definitely think we should try to make this work (in a separate PR)